spec/namespaces/security.yaml

```diff
@@ -629,303 +1057,371 @@ components:
           schema:
             $ref: '../schemas/security._common.yaml#/components/schemas/CreateTenantParams'
       required: true
+    security.create_update_tenancy_config:
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '../schemas/security._common.yaml#/components/schemas/MultiTenancyConfig'
+      required: true
     security.create_user:
       content:
         application/json:
           schema:
             $ref: '../schemas/security._common.yaml#/components/schemas/User'
       required: true
+    security.create_user_legacy:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/security._common.yaml#/components/schemas/User'
+      required: true
+    security.generate_obo_token:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/security._common.yaml#/components/schemas/OBOToken'
+      required: true
     security.patch_action_group:
       content:
-        application/x-ndjson:
+        application/json:
```

Comment:
Isn't the body in the form of NDJSON if the schema type is `array` instead of `object`?

Reply:
[Security APIs accept an actual array not NDJSON](https://opensearch.org/docs/latest/security/access-control/api/#patch-action-groups):
```json
[
  {
    "op": "add", "path": "/CREATE_INDEX", "value": { "allowed_actions": ["indices:admin/create", "indices:admin/mapping/put"] }
  },
  {
    "op": "remove", "path": "/CRUD"
  }
]
```

Reply:
So it's going to be comma-separated JSON?

Reply:
@nhtruong It is valid standards compliant JSON, the top-level type can be an array it doesn't have to be an object. https://stackoverflow.com/a/3833312

Reply:
Gotcha. I was confused by the discrepancy between this and the bulk endpoints in core namespace where NDJSON is used.

Reply:
Yeah those ones are NDJSON which isn't valid JSON because it's not a true JSON array it's just a new object on each line to make it more streaming/memory friendly as you can parse each line independently. That's not the case here though these are true JSON arrays.

Reply:
I'll change it to json

Reply:
These should be JSON.